setup.py

```diff
@@ -162,7 +162,7 @@ def build_extension(self, ext: CMakeExtension):
             destination = os.path.join(os.getcwd(), f"build_{backend}")
             shutil.copy(source, destination)
     
-        if backend in ("lightning_kokkos"):
+        if backend in ("lightning_kokkos", "lightning_qubit"):
```

Comment:
```suggestion
        if backend in ("lightning_kokkos"):
```

Reply:
I don't understand this suggestion. Why skipping `lightning_qubit` in this condition?

Reply:
Ah, my bad! I thought it's redundant